gallery/src/styles/NxCode/NxCodeExample.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+const NxCodeExample = () =>
+  <>
+    <p className="nx-p"><code className="nx-code">&lt;HTML&gt;</code></p>
```

Comment:
Maybe include one of these amidst some other text to show that they are inline

Reply:
> Maybe include one of these amidst some other text to show that they are inline

https://github.com/sonatype/sonatype-react-shared-components/pull/69/commits/23ba6961af8a23886a45e8d3577f9f4ad69b299a